src/main/resources/l10n/JabRef_en.properties

```diff
@@ -2238,10 +2238,24 @@ Copied_version_to_clipboard=Copied_version_to_clipboard
 
 BibTeX_key=BibTeX_key
 Message=Message
+Log_is_copied=Log_is_copied
 
 
 MathSciNet_Review=MathSciNet_Review
 Reset_Bindings=Reset_Bindings
+Detail_information\:=Detail_information\:
+Automatic_Bug_Report-=Automatic_Bug_Report-
+Create_Issue_on_GitHub_is_successful!=Create_Issue_on_GitHub_is_successful!
+Create_Issue_successful!=Create_Issue_successful!
+Your_issue_was_created_in_your_browser!=Your_issue_was_created_in_your_browser!
+The_log_and_exception_information_was_copied_to_your_clipboard.=The_log_and_exception_information_was_copied_to_your_clipboard.
+Please_paste_this_information_(with_Ctrl+V)_in_the_issue_description!=Please_paste_this_information_(with_Ctrl+V)_in_the_issue_description!
+
+
+Developer_information=Developer_information
+Create_Issue=Create_Issue
+*incl._log_and_error_details=*incl._log_and_error_details
+Details_infomation=Details_infomation:
```

Comment:
Please rework some of the language strings keeping in mind that:
- in english you usually write in lower case except at the start of an sentence and proper names such as GitHub. 
- exclamation marks ("!") are rather intusive and shouldn't be used too often, maybe replace some dot (".")
- use the past form if you display if you want to use it in error/info/warning dialogs as something has already happened
- can `Detail_information\:` and `Details_information` be merged?
- the last key and translation don't match